README.md

```diff
@@ -1,12 +1,33 @@
-# Building OpenBLAS
+# OpenBLAS
 
-This is a repository to trigger builds of OpenBLAS on Travis-CI (for aarch64,
-ppc64, s390x) and github actions for all the others.
+We build OpenBLAS on Travis-CI (for linux aarch64, ppc64, s390x) and github actions
+for linux, windows, macOS x86_64 and macOS arm64.
 
-The OpenBLAS libraries get uploaded to
+Tarballs are at
 https://anaconda.org/scientific-python-nightly-wheels/openblas-libs/files
 
-A project using these libraries, for Manylinux or macOS, will need the
+A project using the tarball, for Manylinux or macOS, will need the
 ``gfortran-install`` submodule used here, from
 https://github.com/MacPython/gfortran-install
 
+We also build and upload a pip-installable wheel. The wheel is self-contianed,
+no additional fortran support installations are needed. The wheel supplies
```

Comment:
It may be useful to spell out what this means. I think it is saying that the wheel will include the gfortran runtime? And it's name-mangled so we don't get an issue with that runtime being included in both this wheel and the scipy wheel?

Reply:
The wheels include the needed runtimes. For instance, the macos wheel has
```
scipy_openblas64/.dylibs/libgfortran.5.dylib
scipy_openblas64/.dylibs/libquadmath.0.dylib
scipy_openblas64/.dylibs/libgcc_s.1.1.dylib
scipy_openblas64/lib/libopenblas_python.so
```

and the manylinux2014 one has
```
scipy_openblas64.libs/libquadmath-96973f99.so.0.0.0
scipy_openblas64.libs/libgfortran-040039e1.so.5.0.0
scipy_openblas64/lib/libopenblas_python.so
```
and the windows one is only (it "inherits" the single-dll-statically-linked-to-llibgfortran from the previous openblas-libs configuration)
```
scipy_openblas64/lib/libopenblas64__v0.3.23-293-gc2f4bdbb-gcc_10_3_0.dll
```
So auditwheel does a better job than delocate at name mangling.

Reply:
👍🏼. I'd suggest adding all that to the docs, it's useful info.